release-please-config.json

```
@@ -13,11 +12,11 @@
     "packages/hackathon": { },
     "packages/run-it": { "release-as": "" },
     "packages/sdk": { },
-    "packages/sdk-codegen": { "release-as": "21.0.18" },
-    "packages/sdk-codegen-scripts": { "release-as": "21.0.18" },
+    "packages/sdk-codegen": { "release-as": "21.0.19" },
+    "packages/sdk-codegen-scripts": { "release-as": "21.0.20", "changelog-sections": [{"type": "chore", "section": "Bug Fixes"}] },
```

@jkaster I just saw [your reply](https://github.com/looker-open-source/sdk-codegen/pull/727#issuecomment-872539118) - would you like me to undo this (no changelog entry) or leave it (changelog entry)?

```suggestion
    "packages/sdk-codegen-scripts": { "release-as": "21.0.20" },
```

---

I think we should keep things as you had them and not accommodate my mistake